packages/touchpoint-ui/src/App.tsx

```diff
@@ -99,23 +99,26 @@ const App = forwardRef<AppRef, Props>((props, ref) => {
 
   const input = props.input ?? "text";
 
+  const hangUp = useCallback(() => {
+    sessionStorage.removeItem("nlxActiveVoiceConversationId");
+    if (input === "voice" || input === "voiceMini") {
+      conversationInitialized.current = false;
+      handler.reset({ clearResponses: true });
+      sessionStorage.removeItem("nlxConversationId");
+    }
+  }, [handler, input]);
+  const _onClose = props.onClose;
```

Comment:
Is this set up here just so the hook rule can do its job more easily? This would read terribly if every destructuring we did introduced these underscore variables everywhere.